compose_rl/ppo/callback.py

```diff
@@ -616,42 +633,54 @@ def _interact_with_env(self, batch: dict[str, torch.Tensor]):
         """
         # Determine the number of generating calls we want to make
         # We can have the generate size be greater than the device train microbatch size
```

Comment:
Question for @bcui-db

Why do we break up the generation by batches?
If I understand vLLM correctly,
If you send a batch of data to vLLM, vLLM breaks up the batch into sets it can handle more easily. It'll process the sets of data, then combine the it back together before sending it back.
When integrating evals, [I just chunked all inputs by number of vLLM engines](https://github.com/databricks-mosaic/research-universe/blob/main/orl_eval/orl_eval/orl_eval_callback.py#L113) and sent them thru. It works well and doesn't OOM regardless of how large the eval was (and therefore how big the batch size was).
TLDR: I think we should nuke the batching logic and just let vLLM handle this.